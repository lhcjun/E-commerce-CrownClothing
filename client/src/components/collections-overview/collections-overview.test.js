import React from 'react';
import { shallow } from 'enzyme';
import { CollectionsOverview } from './collections-overview.component';
import CollectionPreview  from '../collection-preview/collection-preview.component';


describe('CollectionsOverview', () => {
    const mockCollections = [ { id: 1 }, { id: 2 } ];
    const wrapper = shallow(<CollectionsOverview collections={mockCollections} />);

    it('should render CollectionsOverview component', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('should render an equal number of CollectionPreview components as the collections prop', () => {
        expect(wrapper.find(CollectionPreview).length).toEqual(mockCollections.length);
    }); 
});