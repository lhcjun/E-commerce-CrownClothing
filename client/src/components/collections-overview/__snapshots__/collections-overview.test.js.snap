// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionsOverview should render CollectionsOverview component 1`] = `ShallowWrapper {}`;
